app/Services/GameService.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+
+namespace App\Services;
+
+
+use App\Models\Expansion;
+use App\Models\Game;
+use App\Models\GameExpansion;
+use App\Models\GameUser;
+use App\Models\User;
+use Nubs\RandomNameGenerator\All;
+
+class GameService
+{
+    public $user;
+    public $game;
+    private $generator;
+
+    public function __construct()
+    {
+        $this->generator = All::create();
+    }
+
+    public function CreateGame($userName, $expansionIds) {
```

Comment:
use `camelCase` for function names

Reply:
I'll change the function names to be camelCase, been a habit of mine lately in C#